CONTRIBUTING.md

```diff
@@ -59,7 +59,7 @@ pip install .
 
 ## Code style
 
-OpenFL uses [black](https://black.readthedocs.io/en/stable/) and [isort](https://pycqa.github.io/isort/) to format the code.
+OpenFL uses [black](https://black.readthedocs.io/en/stable/), [isort](https://pycqa.github.io/isort/) and [precommit](https://pre-commit.com/) to format the code.
 

```

Comment:
Nit: Instead you can mention - `./shell/format.sh` applies all suggested formatting changes, and runs pre-commit checks.

Reply:
changes done in commit d4bcffd6711c481614d4a2e3c7811b36a3db83d0